### ISSUE TITLE
`Missing Property "swagger"` on tsconfig.json

### ISSUE THREAD

**Issue by softcraft-development** [42Crunch/vscode-openapi#9]
As of the latest Visual Studio update, it appears that `vscode-openapi` is attempting to validate my `tsconfig.json` against the swagger format. In the "Problems" pane for my workspace, I now have these messages:

```
{
	"resource": "/Users/craig/Sites/spoonsapp/spoons-api/src/tsconfig.json",
	"owner": "_generated_diagnostic_collection_name_#1",
	"severity": 4,
	"message": "Missing property \"swagger\".",
	"startLineNumber": 1,
	"startColumn": 1,
	"endLineNumber": 1,
	"endColumn": 2
}
```

I have similar messages for missing properties `info` and `path`, as well as `Property {name} is not allowed` for `exclude` and `extends`. 

This wasn't happening before the update, and it disappears when I disable the extension. I'm also getting the same set of messages for `/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/typescript-basics/schemas/tsconfig.schema.json`.

If you need more details and/or config files, let me know, and I'll supply them.

**Comment by ak1394** [comment]
Thanks for reporting it! Fixed in in 1.8.7

**Comment by softcraft-development** [comment]
Looks good! Thanks for the amazingly quick turnaround!
